bundles/framework/myplaces3/service/MyPlacesService.js

```diff
@@ -406,8 +391,8 @@ Oskari.clazz.define('Oskari.mapframework.bundle.myplaces3.service.MyPlacesServic
             place.setLink(Oskari.util.sanitize(properties.link));
             place.setImageLink(Oskari.util.sanitize(properties.image_url));
             place.setCategoryId(properties.category_id);
-            place.setCreateDate(this._formatDate(properties.created));
-            place.setUpdateDate(this._formatDate(properties.updated));
+            place.setCreateDate(Oskari.util.formatDate(properties.created, false));
```

Comment:
I think we should have the date as `Date` in the model and only format when showing it (in the table renderer).